app/models/statistics-store/statistics-supported-day.ts

```diff
@@ -17,18 +17,9 @@ export const StatisticsSupportedDayModel = types
   .props({
     dayID: types.identifier,
     contents: types.array(StatisticsSupportedContentModel),
+    totalCreatorsCount: types.optional(types.number, 0),
   })
   .views(self => ({
-    get totalCreatorsCount() {
-      const creators = new Set<string>()
-      self.contents.forEach(content => {
-        const { info: { creator: { likerID = "" } = {} } = {} } = content
-        if (likerID && !creators.has(likerID)) {
-          creators.add(likerID)
-        }
-      })
-      return creators.size
-    },
```

Comment:
Just curious, how much performance gain do we yield by not using view here?

Reply:
Hard to feel it, but I prefer using the number directly from API response